sapl/sessao/views.py

```diff
@@ -529,6 +529,181 @@ def get_presencas_generic(model, sessao, legislatura):
             yield (m.parlamentar, False)
 
 
+def materias_sessao_plenaria_ajax(request):
+    url = request.GET['url_atual'].split("/")
+    f_url = url[len(url)-1]
+
+    if f_url == "create":
+        f_url_1 = url[len(url)-2]
+        sessao_plenaria_id = url[len(url)-3]
+
+        if f_url_1 == "expedientemateria":
+            materias_sessao = ExpedienteMateria.objects.filter(
+                sessao_plenaria=sessao_plenaria_id
+            )
+        elif f_url_1 == "ordemdia":
```

Comment:
Que nem eu falei com o @cantuariavc, eu não acho legal ficar manipulando URL assim pq isso torna o código bem frágil. Qualquer mudança na URL, ou se a gente tirar o crud to Mazza, por exemplo, pode quebrar tudo. Teria como vc ver uma forma de manter a lógica principal sem precisar fazer esse tipo de operação? 

Um campo hidden, um parâmetro extra no POST/GET, uma variável de sessão, um cookie, qualquer coisa pra evitar esse tipo de manipulação de URL. 😅

Reply:
Alterado. PR #3224.